src/viur/core/prototypes/tree.py

```diff
@@ -830,6 +913,89 @@ def onDeleted(self, skelType: SkelType, skel: SkeletonInstance):
         if user := current.user.get():
             logging.info(f"""User: {user["name"]!r} ({user["key"]!r})""")
 
+    def onClone(self, skelType: SkelType, skel: SkeletonInstance, src_skel: SkeletonInstance):
+        """
+            Hook function that is called before cloning an entry.
+
+            It can be overwritten to a module-specific behavior.
+
+            :param skelType: Defines the type of the node that is cloned.
+            :param skel: The new Skeleton that is being created.
+            :param src_skel: The source Skeleton `skel` is cloned from.
+
+            .. seealso:: :func:`clone`, :func:`onCloned`
```

Comment:
```suggestion
        Hook function that is called before cloning an entry.

        It can be overwritten to a module-specific behavior.

        :param skelType: Defines the type of the node that is cloned.
        :param skel: The new SkeletonInstance that is being created.
        :param src_skel: The source SkeletonInstance `skel` is cloned from.

        .. seealso:: :func:`clone`, :func:`onCloned`
```